components/camera/videosource/webcam.go

```diff
@@ -236,6 +238,13 @@ func getLabelFromCamera(cam camera.Camera, logger golog.Logger) string {
 func NewWebcamSource(ctx context.Context, name string, attrs *WebcamAttrs, logger golog.Logger) (camera.Camera, error) {
 	cam, err := findAndMakeCamera(ctx, attrs, attrs.Path, logger)
 	if err != nil {
+		// If we are on an Orin AGX, we need to check if the daughterboard and the camera driver are installed.
+		if runtime.GOOS == "linux" && runtime.GOARCH == "arm64" {
```

Comment:
It seems like this should be specific to a model and if it is for webcams, then done at init time? Otherwise we will set a precedent for doing multiple checks here.